CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   change aims to streamline the code and eliminate unnecessary components.
 
 ### Changed
+- Fixed `s` update in `DLSolver._solve()` in ccvm_simulators.solvers.dl_solver.py
 - Streamlined README by relocating and optimizing architecture diagrams.
```

Comment:
```suggestion
### Fixed
- Fixed issue where `s` was being updated incorrectly on each iteration of `DLSolver._solve()`

### Changed
- Streamlined README by relocating and optimizing architecture diagrams.
```

Reply:
Let's add a new section for bug fixes and put this one in it!

Reply:
I added a section and inserted the change as suggested.